Program.cs

```diff
@@ -5,11 +5,91 @@ namespace Calculator
   public static class Program
   {
     private static void Main(string[] args)
+    {
+      PrintWelcomeMessage();
+
+      while (true)
+      {
+        PerformOneCalculation();
+      }
+    }
+
+    private static void PrintWelcomeMessage()
     {
       Console.WriteLine("Welcome to the calculator!");
       Console.WriteLine("==========================");
+    }
+
+    private static void PerformOneCalculation()
+    {
+      var op = AskForOperator();
+      var n = AskForNumberArray(op);
+      var a = CalculateAnswer(op, n);
```

Comment:
These variable names aren't very descriptive making it a little hard to remember exactly what they mean when they're used later (e.g. `n` is often used for a single integer but in this case its a list). I would recommend using full words to describe most variables (unless they are things like `i` in a for loop etc) to make the code clearer for outside readers :D

Reply:
Also since we know what these values are going be you can change them from `var` to e.g. `int[]`